css/css-images/image-set/image-set-parsing.html

```diff
@@ -5,26 +5,42 @@
 <meta name="assert" content="General image-set parsing  follows CSS Images 4 rules.">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<script src="support/image-set-parsing-tests.js"></script>
+<script src="/css/support/parsing-testcommon.js"></script>
 <body>
 <script>
-function check_image_set(tst) {
-  var div = document.createElement('div');
-  div.setAttribute("style", `${tst.property}: image-set(${tst.imageSet})`)
-
-  var inline_style = div.style.getPropertyValue(tst.property);
-  assert_equals(inline_style.startsWith('image-set'), tst.valid);
-
-  document.body.appendChild(div);
-  var computed_style = getComputedStyle(div).getPropertyValue(tst.property);
-  assert_equals(computed_style.startsWith('image-set'), tst.valid);
+function test_valid_value_variants(property, value, serializedValue) {
+  test_valid_value(property, value, serializedValue);
```

Comment:
maybe:

```
if (serializedValue === undefined)
  serializedValue  = value;
```

Would allow you to simplify some of the tests.

Reply:
Yeah, I debated actually making this change in the shared JS (which currently uses `arguments.length < 3`), but I ultimately felt like that should be a different change.